sqlserver/requirements.txt

```diff
@@ -2,7 +2,4 @@
 adodbapi==2.6.0.7
 pyro4==4.36
 
-
-# optional - SDK question here
-# Requires unixodbc on Linux
```

Comment:
Note to self: fix the requirement-checking tool to deal with the `sys_platform` specification in the pip requirements file.